grafana/jellyfish-server-overiew.json

```diff
@@ -0,0 +1,758 @@
+{
+    "annotations": {
+        "list": [
+            {
+                "builtIn": 1,
+                "datasource": {
+                    "type": "grafana",
+                    "uid": "-- Grafana --"
+                },
+                "enable": true,
+                "hide": true,
+                "iconColor": "rgba(0, 211, 255, 1)",
+                "name": "Annotations & Alerts",
+                "type": "dashboard"
+            }
+        ]
+    },
+    "editable": true,
+    "fiscalYearStartMonth": 0,
+    "graphTooltip": 0,
+    "id": 1,
+    "links": [],
+    "liveNow": false,
+    "panels": [
+        {
+            "datasource": {
+                "type": "prometheus",
+                "uid": "PBFA97CFB590B2093"
+            },
+            "description": "",
+            "gridPos": {
+                "h": 13,
+                "w": 8,
+                "x": 0,
+                "y": 0
+            },
+            "id": 4,
+            "links": [
+                {
+                    "targetBlank": true,
+                    "title": "Jellyfish docs",
+                    "url": "https://jellyfish-dev.github.io/jellyfish-docs/"
+                }
+            ],
+            "options": {
+                "code": {
+                    "language": "plaintext",
+                    "showLineNumbers": false,
+                    "showMiniMap": false
+                },
+                "content": "This is Jellyfish dashboard\n\n![Text](https://avatars.githubusercontent.com/u/119447297?s=400&u=f714adece660ae05d4b177d1d4e7d7886aa4e9d5&v=4)",
```

Comment:
Based on the last commit message seems like you wanted to delete those panels :smile: Seems like you forgot to save before exporting dashboard